scripts/install/opt/seagate/cortx/motr/libexec/motr_data_recovery.sh

```diff
@@ -1,4 +1,23 @@
 #!/usr/bin/env bash
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
Copyright (c) 2020 - 2021 Seagate Technology LLC?

Reply:
handled in commit ac81173.